test/CMakeLists.txt

```diff
@@ -446,6 +446,20 @@ if (TARGET TBB::tbb)
     tbb_add_test(SUBDIR tbb NAME test_eh_thread DEPENDENCIES TBB::tbb)
     tbb_add_test(SUBDIR tbb NAME test_global_control DEPENDENCIES TBB::tbb)
     tbb_add_test(SUBDIR tbb NAME test_task DEPENDENCIES TBB::tbb)
+
+    if (TBB_FUZZ_TESTING)
+        if (NOT CMAKE_CXX_COMPILER_ID STREQUAL IntelLLVM)
+            message(FATAL_ERROR "Fuzzing requires IntelLLVM compiler.")
+        endif()
```

Comment:
Clang is applicable here as well.

Reply:
It is also supported by MSVC.
https://learn.microsoft.com/en-us/cpp/build/reference/fsanitize?view=msvc-170

Reply:
I will add Clang. MSVC support, however, is experimental and incomplete. In fact, since this is a special test option, I will enable it only on Linux with IntelLLVM and Clang.

Reply:
Done.